core-blocks/file/editor.scss

```diff
@@ -4,6 +4,10 @@
 	align-items: center;
 	margin-bottom: 0;
 
+	> div {
```

Comment:
Maybe we can add a class to this div something like wp-block-file__content-wrapper so we can target it using the class name instead of "> div" which is more vulnerable to a regression in a future change.

Reply:
This sounds much better. Addressed in [this commit](https://github.com/WordPress/gutenberg/pull/8303/commits/cf350592d5c30ac1fd7681c8a28357627f651950). Thank you ❤️